tests/http/src/test/resources/http-request-query-params-config.xml

```diff
@@ -47,8 +47,8 @@
         <httpn:request config-ref="config" path="testPath"  key="ANY">
             <httpn:request-builder>
                 <!-- Not currently supported. -->
-                <!--<httpn:requester-request-builder queryParams="#[{'testName1':null, 'testName2': [null]}]"/>-->
-                <httpn:requester-request-builder queryParams="#[{'testName1':null, 'testName2': null}]"/>
+                <!--<httpn:simple-request-builder queryParams="#[{'testName1':null, 'testName2': [null]}]"/>-->
```

Comment:
I'm keeping the lines commented to have a clear idea of what we should test once we support query params collections.